docx_editor_mcp/tools.py

```
@@ -0,0 +1,824 @@
+        return cast(dict[str, Any], result)
+    cached = result
+
+    result = {"success": True, "path": cached.path}
+
+    if cached.dirty:
+        result["warning"] = "Document had unsaved changes that were discarded."
+
+    # Close the document
+    try:
+        cached.document.close()
+    except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)